test/built-ins/Temporal/PlainDateTime/prototype/with/cases.js

```diff
@@ -0,0 +1,77 @@
+// Copyright (C) 2022 Igalia, S.L. All rights reserved.
```

Comment:
Same comment as above about the name, I'd recommend `basic.js`

Reply:
done